Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPINamespaceCocoa.mm

```diff
@@ -220,6 +230,28 @@
     return *m_scripting;
 }
 
+#if ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
+WebExtensionAPISidebarAction& WebExtensionAPINamespace::sidebarAction()
+{
+    // Documentation: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/sidebarAction
```

Comment:
nit: can drop the en-US
// Documentation: https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sidebarAction